sechub-pds/src/main/java/com/mercedesbenz/sechub/pds/security/PDSSecurityConfiguration.java

```diff
@@ -53,24 +53,27 @@ public class PDSSecurityConfiguration extends AbstractAllowPDSAPISecurityConfigu
     public UserDetailsService userDetailsService() {
         /* @formatter:off */
         UserDetails user =
-             User.builder()
-                .username(techUserId)
-                .password(techUserApiToken)
-                .roles(PDSRoles.USER.getRole())
-                .build();
+                User.builder()
+                        .username(techUserId)
+                        .password(disablePasswordEncode(techUserApiToken))
+                        .roles(PDSRoles.USER.getRole())
+                        .build();
         /* remove field after start */
-        techUserApiToken=null;
+        techUserApiToken = null;
```

Comment:
Just a reminder: Please do always execute `./gradlew spotlessApply` to automatically format source code before you push to Git.